### PR TITLE
Bower support, fixes #39

### LINKED ISSUE #39: Bower support
Bower is popular and easy to use. To install elemental using bower try: bower install elementalui

### PR DESCRIPTION
**Pull request by nkbt**
Script for automatic publishing to Bower.

For now OSX/Linux only

To test, I checked out `v0.5.1` tag, applied patch with these changes and run script.

### CODE REVIEW

File: bower.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+  "version": "0.0.0",
+  "description": "React UI Framework",
+  "main": "elemental.js",
+  "license": "MIT",
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/elementalui/elemental-dist.git"
+  },
+  "dependencies": {
+    "react": "^0.14"
```
Comment (nkbt) [inline bower.json]: There is a problem with bower dependencies. `react-dom` and `react-addons-css-transition-group` are not published in bower registry, so I did not add them here.
It is probably possible to set those deps directly to github

File: scripts/bower.sh [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+echo REPO=$REPO
+echo VERSION=$VERSION
+
+pushd .
+rm -rf /tmp/$ORG/$REPO
+mkdir -p /tmp/$ORG/$REPO
+git clone git@github.com:$ORG/$REPO.git /tmp/$ORG/$REPO
+npm run build
+cp -f ./dist/* /tmp/$ORG/$REPO
+cp -f ./bower.json /tmp/$ORG/$REPO
```
Comment (nkbt) [inline scripts/bower.sh]: We override `bower.json` so it is possible to change it in the main repo here and never touch `elemental-dist`.

File: scripts/bower.sh [hunk 3]
```diff
@@ -0,0 +1,25 @@
...
+cd /tmp/$ORG/$REPO
+git add .
+git commit -m "Release $VERSION"
+bower version $VERSION
```
Comment (nkbt) [inline scripts/bower.sh]: `bower version` overrides version written in bower.json file (similar to npm) so there is no need to update it manually. This is why version is set to `0.0.0`

File: bower.json [hunk 4]
```diff
@@ -0,0 +1,35 @@
+{
+  "name": "elemental",
+  "version": "0.0.0",
```
Comment (nkbt) [inline bower.json]: See comment below on why version is `0.0.0`

**Comment by JedWatson** [comment]
:+1: looks great @nkbt thanks!

**Pull request merged**